### PR TITLE
Block grid boxes

### PR DESCRIPTION
**Pull request by sineade**
:art:
Adding in new foundation code to utilize their block grid for the blue boxes we use all over my account.

### CODE REVIEW

File: app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss [hunk 1]
```diff
@@ -91,3 +94,40 @@
...
+  }
+}
+
+.icon-container{
+  width: 100px;
+  height: 80px;
+  margin: $base-margin*4 auto 0 auto;
+  background-repeat: no-repeat;
+  background-position: center center;
+  width: auto;
```
Comment (MathieuGilbert) [inline app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss]: Is this overriding the `width: 100px` on line 116?

**Comment by MathieuGilbert** [comment]
:hankey:

File: app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss (hunk 1 above)
Comment (sineade) [inline app/assets/stylesheets/ama_layout/layout_components/blue-boxes.scss]: oh ya, it was. Removing that now.

**Comment by sineade** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:art: :art: :art:

**Pull request merged**